packages/cover-traffic-daemon/metrics.md

```diff
@@ -0,0 +1,53 @@
+On every tick of cover-traffic strategy, a few metrics are saved in the persisted state (`packages/cover-traffic-daemon/ct.json`).
+
+Other metrics can be deduced from logs. Note that logs start with `hopr:cover-traffic `
+
+1. At each tick, a log lists the current balance of the CT node, channels to open and to close.
+   ```sh
+   strategy tick: ${Date.now()} balance:${balance.toString()} open:${toOpen
+           .map((p) => p[0].toPeerId().toB58String())
+           .join(',')} close: ${toClose.map((p) => p.toPeerId().toB58String()).join(',')}`.replace('\n', ', ')
+   ```
+2. Status of all the CT channels registered in the persisted state:
+   ```sh
+   'channels',
+   ctChannels.map((c) => `${c.destination.toB58String()} - ${c.latestQualityOf}, ${c.openFrom}`).join('; ')
```

Comment:
How does the output look like? I can imagine it by seeing the code but I would be easier to see some sample output